### PR TITLE
Fix typos in tidy-evaluation.Rmd

### PR DESCRIPTION
**Pull request by gvelasq2**

### CODE REVIEW

File: vignettes/tidy-evaluation.Rmd [hunk 1]
```diff
@@ -78,9 +78,9 @@ is `dplyr::select()`). As this method is relying on the R interpreter,
 the grammar is much more likely to behave like real R code.
 
 R DSLs are traditionally implemented with a mix of both principles.
-Expressions are parsed in ad hoc ways, but are eventually evaluated in
```
Comment (lionel-) [inline vignettes/tidy-evaluation.Rmd]: do you italicize latin words in your documents @hadley ?

Comment (hadley) [inline vignettes/tidy-evaluation.Rmd]: Nope

**Comment by gvelasq2** [comment]
Thanks @hadley and @lionel-. I removed the italics.

File: vignettes/tidy-evaluation.Rmd [hunk 2]
```diff
@@ -169,7 +169,6 @@ expression in turn, but how can we combine all expressions into one
 and evaluate it tidily at once? An expression can only be evaluated in
 a single environment. This is where quosures come into play.
 
-
```
Comment (lionel-) [inline vignettes/tidy-evaluation.Rmd]: oops, can you also leave 2 newlines before headings please

**Comment by gvelasq2** [comment]
Thanks @lionel-. All set.

**Comment by lionel-** [comment]
Thanks!

**Pull request merged**